core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -709,6 +709,12 @@ private[spark] class TaskSetManager(
         }
         ef.exception
 
+      case e: ExecutorLostFailure =>
+        if (e.isNormalExit) {
```

Comment:
this actually masks error messages if the executor did not exit normally. We need something like
```
case e: ExecutorLostFailure if e.isNormalExit =>
```